src/doc/gui_manual/WorkingWithFiles/Read_Write_Options.rst

```diff
@@ -0,0 +1,184 @@
+.. _Read_Write_Options:
+
+Database Read and Write Options
+-------------------------------
+Several database plugins have options that effect reading and/or writing with that
+format. These are described in this section. Alternatively, in some cases, the
+behavior of a database plugin may be effected by enviornment variables.
+
+Chombo
```

Comment:
The resulting formatting throws me off a little. So when you have this, do you mean that the property "Detect Compound Variables" is the same for both Chombo and Exodus? Or is this a stub for the chombo database to be filled out later?

Reply:
Well, its because I left some sections blank but didn't make that clear. I've added some boilerplate text in those blank sections to help clarify.